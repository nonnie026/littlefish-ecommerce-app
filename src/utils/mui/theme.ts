import { createTheme } from '@mui/material/styles'

const theme = createTheme({
  typography: {
    fontFamily: ['Inter', 'sans-serif'].join(',')
  },
  components: {
    MuiCard: {
      styleOverrides: {
        root: {
          boxShadow: 'none',
          height: '100%'
        }
      }
    }
  }
})

export default theme
