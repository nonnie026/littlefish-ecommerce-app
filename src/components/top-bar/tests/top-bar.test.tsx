import { ThemeProvider } from '@mui/material/styles'
import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import { theme } from '../../../utils/mui/theme'
import { TopBar } from '../index'

describe('TopBart', () => {
  const view = () =>
    render(
      <ThemeProvider theme={theme}>
        <TopBar />
      </ThemeProvider>
    )

  afterEach(() => {
    jest.clearAllMocks()
  })

  it('renders snapshot', () => {
    const { asFragment } = view()

    expect(asFragment()).toMatchSnapshot()
  })
})
