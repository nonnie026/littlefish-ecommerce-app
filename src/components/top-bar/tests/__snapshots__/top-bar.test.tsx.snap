// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBart renders snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-mtkkfu"
  >
    <div
      class="MuiBox-root css-1axa72v"
    >
      <div
        class="MuiGrid-root css-19fssl2-MuiGrid-root"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-1yxn50p-MuiTypography-root"
        >
          Mon - Thu: 
          <strong>
            9:00 AM - 5:30 PM
          </strong>
        </p>
        <p
          class="MuiTypography-root MuiTypography-body1 css-1yxn50p-MuiTypography-root"
        >
          Visit our showroom in 1234 Street Adress City Address, 1234 
          <strong
            color="white"
          >
            Contact us
          </strong>
        </p>
        <p
          class="MuiTypography-root MuiTypography-body1 css-1yxn50p-MuiTypography-root"
        >
          <strong>
            (00) 1234 5678
          </strong>
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
